aidies-project/routes/api/auth.js

```diff
@@ -0,0 +1,52 @@
+const jwt = require("jsonwebtoken");
+const uuid = require('uuid').v4;
+const { jwtConfig } = require('../../config/index');
+const { User } = require('../../db/models');
+
+const { secret, expiresIn } = jwtConfig;
+
+const getUserToken = (user) =>{
+    const userDataForToken = user.toSafeObject();
+    const jwtid = uuid();
+    const token = jwt.sign(
+        { data: userDataForToken },
+        secret,
+        { expiresIn: Number.parseInt(expiresIn, 10) }
+    );
+    return { jti: jwtid, token: token };
```

Comment:
You can also remove this `jwtid` and have this `getUserToken` function just return a token, since it doesn't look like we're using the `jwtid` information anywhere.
```suggestion
    const token = jwt.sign(
        { data: userDataForToken },
        secret,
        { expiresIn: Number.parseInt(expiresIn, 10) }
    );
    return token;
```